CDTDatastore/Encryption/CDTEncryptionKeySimpleProvider.m

```diff
@@ -26,7 +26,13 @@ @interface CDTEncryptionKeySimpleProvider ()
 
 @implementation CDTEncryptionKeySimpleProvider
 
-- (instancetype)init { return [self initWithKey:nil]; }
+- (instancetype)init
+{
+#pragma clang diagnostic push
```

Comment:
Wouldn't it be easier and clearer to just implement this as `return nil;` as it's marked unavailable?